server/services/Crud.js

```diff
@@ -0,0 +1,258 @@
+const _ = require('lodash');
+class Crud {
```

Comment:
I don't really see the point of this class, it isn't extending the functionality of mongoose so why not just use the built in methods. It only makes sense to add a layer of abstraction if you need to extend functionality in some way